website/docs/r/cse_insights_status.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cse_insights_status"
+description: |-
+  Provides a CSE Insights Status
+---
+
+# sumologic_cse_insights_status
+Provides a CSE Insights Status.
```

Comment:
"CSE" -> "Sumo Logic CSE"

Reply:
Can we add more details here?

Reply:
done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/274/commits/d6c3b409bfda3ea68dbdf9e0034757c6ae2cee19